docs-next/src/content/docs/explainers/compilers-deprecation.mdx

```diff
@@ -3,15 +3,15 @@ description: Compilers deprecation notice.
 title: Compilers deprecation
 ---
 
-If you're here, you probably hit the deprecation notice.  Sorry about that!
```

Comment:
Have we built and reviewed the site after formatting? My understanding is Prettier doesn't support MDX and may mess up some spacing. Not seeing anything alarming yet, just want to call it out